sklearn/mixture/tests/test_gaussian_mixture.py

```diff
@@ -1387,3 +1389,30 @@ def test_gaussian_mixture_single_component_stable():
     X = rng.multivariate_normal(np.zeros(2), np.identity(2), size=3)
     gm = GaussianMixture(n_components=1)
     gm.fit(X).sample()
+
+
+def test_gaussian_mixture_all_init_does_not_estimate_gaussian_parameters(monkeypatch):
```

Comment:
Nit:
We have a global_random_seed fixture which can run tests on seeds varying from 0 to 99. I checked on my local system that all these seeds pass. (We need to set the SKLEARN_TESTS_GLOBAL_RANDOM_SEED=all with a specific test to run all the seeds.)
```suggestion
def test_gaussian_mixture_all_init_does_not_estimate_gaussian_parameters(
    monkeypatch,
    global_random_seed,
):
```